Sources/Core/Needs Conversion/Settings.swift

```diff
@@ -42,15 +44,26 @@ class Settings: SettingsManaging {
   // TODO: Figure out where this was coming from. Pretty sure it's tied to logging
   let graphApiDebugParameter: GraphApiDebugParameter = .none
 
-  // TODO: probably should not be settable from everywhere but should come from some sort of config
-  static var graphAPIVersion: String = ""
+  /**
+   Overrides the default Graph API version to use with `GraphRequest`s. This overrides the provided default.
+   */
+  var graphAPIVersion = GraphAPIVersion(major: 3, minor: 4)
```

Comment:
Current Graph API version is `v3.2`

```suggestion
  var graphAPIVersion = GraphAPIVersion(major: 3, minor: 2)
```